resources/views/partials/navbar.blade.php

```diff
@@ -58,14 +58,14 @@ class="bg-gray-800 flex text-sm rounded-full focus:outline-none focus:ring-2 foc
                 id="user-menu-button" aria-expanded="false" aria-haspopup="true">
                 <span class="sr-only">Open user menu</span>
                 <img class="h-8 w-8 rounded-full"
-                  src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
+                  src="{{Auth::user()->avatarPath}}"
```

Comment:
```suggestion
                  src="{{ Auth::user()->avatarPath }}"
```
Stay with convention